src/de/grobox/liberario/TransportNetworks.java

```diff
@@ -579,4 +610,12 @@ TransportNetwork getTransportNetwork(String idString) {
 	private String getString(int res) {
 		return context.getString(res);
 	}
+
+	private String region(String name, String flag) {
+		return flag + " " + name;
+	}
+
+	private String description(String desc, String networks) {
```

Comment:
Did you try using the `@StringRes` annotation for the parameter like I had said? The text expansion should then work as before. Also, looks like you forget to call `getString()` now. Did you test your latest commits?